jsonschema-kafkaconnect-converter/src/main/java/com/amazonaws/services/schemaregistry/kafkaconnect/jsonschema/typeconverters/TypeConverterFactory.java

```diff
@@ -50,7 +50,12 @@ public TypeConverter get(@NonNull org.everit.json.schema.Schema jsonSchema,
         if (jsonSchema instanceof BooleanSchema) {
             typeConverter = get(Schema.Type.BOOLEAN);
         } else if (jsonSchema instanceof NumberSchema) {
-            typeConverter = get(Schema.Type.valueOf(connectType.toUpperCase()));
+            // If no connect type passed then assume that connect schema is for FLOAT64 type data
+            if (connectType == null) {
+                typeConverter = get(Schema.Type.valueOf("FLOAT64"));
```

Comment:
Minor: Can we add metadata similar to https://github.com/awslabs/aws-glue-schema-registry/blob/kafka-connect-converter-protobuf/protobuf-kafkaconnect-converter/README.md ?